packages/microsoft-reactnative-sampleapps/index.windows.js

```diff
@@ -38,6 +38,12 @@ var getCallback = function(prefix) {
   };
 };
 
+var getErrorCallback = function(prefix) {
+  return function(error) {
+    log(prefix + (error || {}).message);
```

Comment:
>|| {} [](start = 24, length = 5)

What is the motivation for this change?

Reply:
The error object has a member 'message' the ||{} technique is used to ensure that we do not crash should the error be null.